src/machine/machinectl.c

```diff
@@ -1456,6 +1456,7 @@ static int get_settings_path(const char *name, char **ret_path) {
 static int edit_settings(int argc, char *argv[], void *userdata) {
         _cleanup_(edit_file_context_done) EditFileContext context = {
                 .remove_parent = false,
+                .overwrite_with_origin = false,
```

Comment:
We usually not list 0, NULL, or false in initializes. Please drop the two, and simply use `{}`.